boards/riscv/icev_wireless/icev_wireless-pinctrl.dtsi

```diff
@@ -31,6 +31,31 @@
 		};
 	};
 
+	spim2_sleep: spim2_sleep {
+		group1 {
+			pinmux = <ESP32_PINMUX(4, ESP_NOSIG, ESP_NOSIG)>,
+				 <ESP32_PINMUX(5, ESP_NOSIG, ESP_NOSIG)>,
+				 <ESP32_PINMUX(6, ESP_NOSIG, ESP_NOSIG)>,
+				 <ESP32_PINMUX(7, ESP_NOSIG, ESP_NOSIG)>;
+		};
+	};
+
+	spim2_cs_as_gpio: spim2_cs_as_gpio {
+		group1 {
+			pinmux = <SPIM2_MISO_GPIO4>,
+				 <SPIM2_SCLK_GPIO5>;
+		};
+		group2 {
+			pinmux = <SPIM2_MOSI_GPIO7>;
+			output-low;
+		};
+		group3 {
+			pinmux = <ESP32_PINMUX(6, 6, 6)>;
```

Comment:
I imagine this is to "unroute" CS so that GPIO API can be used, right? If so, properties (e.g. drive-open-drain) should not be required as they'll be controlled by the GPIO API.